app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/ContributorsViewModel.java

```diff
@@ -50,6 +61,15 @@ public void onClickContributor(String htmlUrl) {
         }
     }
 
+    @Bindable
```

Comment:
I think it's not good to check by `contributors`. How about adding `setter` and `getter`?
Like this

```java
private int loadingVisibility;

@Bindable
public int getLoadingVisibility() {
    return loadingVisibility;
}

public void setLoadingVisibility(int visibility) {
    this.loadingVisibility = visibility;
    notifyPropertyChanged(BR.loadingVisibility);
}
```

```
       return contributorsRepository.findAll().map(contributors -> {
            this.contributors = contributors;
            setLoadingVisibility(View.VISIBLE);
```